### PR TITLE
OptionType.NUMBER support

### PR DESCRIPTION
**Pull request by Sanduhr32**

### CODE REVIEW

File: src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt [hunk 1]
```diff
@@ -56,6 +56,7 @@ inline fun <reified T> SubcommandData.option(name: String, description: String,
 
 fun OptionData.choice(name: String, value: String) = addChoice(name, value)
 fun OptionData.choice(name: String, value: Int) = addChoice(name, value)
+fun OptionData.choice(name: String, value: Float) = addChoice(name, value)
```
Comment (Sanduhr32) [inline src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt:59]:
```suggestion
fun OptionData.choice(name: String, value: Double) = addChoice(name, value)
```
if wanted / needed

File: src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt [hunk 2]
```diff
@@ -56,6 +56,7 @@ inline fun <reified T> SubcommandData.option(name: String, description: String,
 
 fun OptionData.choice(name: String, value: String) = addChoice(name, value)
 fun OptionData.choice(name: String, value: Int) = addChoice(name, value)
```
Comment (Sanduhr32) [inline src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt:58]:
```suggestion
fun OptionData.choice(name: String, value: Long) = addChoice(name, value)
```
also possible if wanted / needed

File: src/main/kotlin/dev/minn/jda/ktx/interactions/utils.kt [hunk 3]
```diff
@@ -20,6 +20,7 @@ import net.dv8tion.jda.api.entities.*
 import net.dv8tion.jda.api.interactions.commands.OptionType
 
 inline fun <reified T> optionType() = when(T::class.java) {
+    java.lang.Float::class.java, java.lang.Double -> OptionType.NUMBER
```
Comment (MinnDevelopment) [inline src/main/kotlin/dev/minn/jda/ktx/interactions/utils.kt:23]:
```suggestion
    java.lang.Float::class.java, java.lang.Double::class.java -> OptionType.NUMBER
```

File: src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt (hunk 1 above)
Comment (MinnDevelopment) [reply src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt:59]: Make it double

File: src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt (hunk 2 above)
Comment (MinnDevelopment) [reply src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt:58]: making it long is probably fine

**Pull request merged**